eclcli/monitoring/monitoringclient/v2/shell.py

```diff
@@ -16,10 +16,17 @@
 
 import argparse
 import functools
-import json
```

Comment:
What is the reason for removing `import json` ?
The removal caused an `unresolved reference 'json'`.

Reply:
Oh... 🤦‍♂It seems that I accidentally deleted it when I made other corrections.
I'll restore it.